src/test/java/bit/javaoop/CEOTest.java

```diff
@@ -0,0 +1,32 @@
+package bit.javaoop;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CEOTest {
+    @Test
+    void hireManager() {
+        CEO ceo = new CEO("Adam");
+        Manager manager = new FixedEmployeeNumberManager("One", 1200, 12);
+        ceo.hireManager(manager);
+        ceo.hireManager(new FixedBudgetManager("Two", 10, 100));
+        assertEquals(2, ceo.getManagersList().size());
+        assertEquals("One", ceo.getManagersList().get(0).getName());
+        assertEquals("Two", ceo.getManagersList().get(1).getName());
+
+    }
+
+    @Test
+    void tostring() {
```

Comment:
toString() jak już ale tak swoją drogą to konwencja nazewnictwa i schemat testów powinno być tak, że jeden test testuje jedną rzecz i nazwa mówi o tym co dokładnie testuje nazwa na wzór shouldCosZrobic.

I jeśli powtarzasz jakiś kawałek kodu na początku każdej metody tak jak tutaj jest to `CEO ceo = new CEO("Adam");` 
powinno to być wyekstrahowane do metody setUp oznaczonej adnotacją @BeforeMethod ta metoda wtedy będzie wołana przed każdym testem.
+ konwencja `//given //when //then`

Po poprawkach wyglądałoby to tak:

![image](https://user-images.githubusercontent.com/18233796/37933782-91ce73a6-314c-11e8-86bf-f2cb98795edb.png)
![image](https://user-images.githubusercontent.com/18233796/37933794-99f323f6-314c-11e8-945b-d1c3d1747451.png)
Prawda, że dużo ładniej 😄